pkgs/tools/system/retry/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib, stdenv, fetchFromGitHub, libtool, automake, autoconf, which, txt2man }:
```

Comment:
Usually everything is indented by two spaces in nixpkgs, but i don't know if that's a PR blocker.

You can use nixpkgs-fmt to format this for you, it'll save you a lot of time